pilot/pkg/networking/util/util.go

```diff
@@ -68,8 +68,9 @@ const (
 	PassthroughFilterChain = "PassthroughFilterChain"
 
 	// Inbound pass through cluster need to the bind the loopback ip address for the security and loop avoidance.
-	InboundPassthroughClusterIpv4 = "InboundPassthroughClusterIpv4"
-	InboundPassthroughClusterIpv6 = "InboundPassthroughClusterIpv6"
+	InboundPassthroughClusterIpv4      = "InboundPassthroughClusterIpv4"
+	InboundPassthroughClusterIpv6      = "InboundPassthroughClusterIpv6"
+	InboundPassthroughClusterDualStack = "InboundPassthroughClusterDualStack"
```

Comment:
```suggestion
	InboundPassthroughClusterDualStack = "InboundPassthroughCluster"
```

Just a nit, given long term we will only have this option it will be cleaner to just not have it named